### PR TITLE
Reorganize content and fix mobile menu

### PR DESCRIPTION
**Pull request by zacharyliu**
Reorganize logistics content to more closely match [site design doc](https://docs.google.com/document/d/1S-zB3jOo5RBu5_p6Muh_HDyQ2OYc3MKKyyD6uhWBORE/edit#).

Fix main dropdown menu behavior on mobile.

### CODE REVIEW

File: _includes/menu.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <nav class="fixed-top full-width">
     <div class="title-bar" data-responsive-toggle="main-menu" data-hide-for="large">
-        <a href="#home"><img src="{{ "img/logo.svg" | prepend: site.baseurl }}?{{site.time | date: '%s%N'}}"></a>
+        <a href="/"><img src="{{ "img/logo.svg" | prepend: site.baseurl }}?{{site.time | date: '%s%N'}}"></a>
```
Comment (casey-chow) [inline _includes/menu.html:3]: Should this be `/#home`? Haven't tested.

**Review by casey-chow — APPROVED** [review]
lgtm

File: _includes/menu.html (hunk 1 above)
Comment (zacharyliu) [reply _includes/menu.html:3]: Hm, guess it depends if you think this should always go to the top of the page. As it is now, this logo just goes to the home page, and if you're already there it does nothing. I don't feel strongly about it either way.

Comment (zacharyliu) [reply _includes/menu.html:3]: Actually, I guess it refreshes the page, so it always goes to the top. So I'd prefer keeping it "/" so the URL is clean.

Comment (casey-chow) [reply _includes/menu.html:3]: sgtm

**Pull request merged**